assignment/core/src/main/java/com/assignment/core/models/BreadcrumbModel.java

```diff
@@ -0,0 +1,72 @@
+package com.assignment.core.models;
+
+import com.day.cq.wcm.api.Page;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.*;
+import org.apache.sling.models.annotations.injectorspecific.*;
+
+import java.awt.desktop.ScreenSleepEvent;
+import java.util.*;
+
+@Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class BreadcrumbModel {
+
+    @SlingObject
```

Comment:
Why did you created a ResourceResolver?

Reply:
Before i wanted to use page manager so to get page manager instance i have created the resource resolver but i used resource based approach , but i forgo to remove resource resolver from the code and in my new commit no.6 i used resource resolver for using page manager Apis